apps/nextjs/src/app/api/aila-download/route.ts

```diff
@@ -115,6 +115,7 @@ async function getHandler(req: Request): Promise<Response> {
       gdriveFileId: fileId,
       userId,
     },
+    cacheStrategy: { ttl: 60 * 5, swr: 60 * 2 },
```

Comment:
The pattern is the to add a TTL and SWR setting to each request that looks cacheable